vscode/webviews/chat/cells/contextCell/ContextCell.tsx

```diff
@@ -100,18 +102,33 @@ export const ContextCell: FunctionComponent<{
                     excludedAtContext: excludedAtContext.length,
                 },
             })
-        }
+        }, [])
 
         const {
             config: { internalDebugContext },
         } = useConfig()
 
+        const telemetryRecorder = useTelemetryRecorder()
+        const oneboxEnabled = useExperimentalOneBox()
+        const logValueChange = useCallback(
+            (value: string | undefined) => {
+                if (oneboxEnabled) {
+                    telemetryRecorder.recordEvent(
+                        'onebox.contextDrawer',
+                        value ? 'expanded' : 'collapsed'
+                    )
```

Comment:
We could combine this into one event Name and store the `expanded` and `collapsed` into the metadata, not sure if there is huge value in storing this as two different events. 

```suggestion
telemetryRecorder.recordEvent(
  'onebox.contextDrawer',
  'clicked',
  {
    [value ? 'expanded' : 'collapsed']: 1
  }
)
```

Reply:
done